bubo/pindora_api.py

```diff
@@ -0,0 +1,49 @@
+import requests
+import json
+from bubo.config import Config

```

Comment:
```suggestion
import json

from bubo.config import Config
```

See "Imports should be grouped in the following order:" - https://pep8.org/#imports